### DISCUSSION TITLE
Does it compress the values?

### DISCUSSION THREAD

**Discussion by BrodaNoel** [react-native-async-storage/async-storage#931] [Q&A]
Hello everyone!
I'm downloading a JSON file which the size is 4MB uncompressed, but just 1mb if it's gzip compressed.

Do you know if this library or if SQLite or whatever in the flow compress the data?

If I am storing those 4MB of text, are they going to actually cost 4MB? Or will be compressed and only cost 1MB?

Thanks!

PS.: I'm not trying to compress any data. I just want to know if it compress or not, in order to properly calculate some app sizes changes regarding the data I store

**Comment by Chris-Imade** [comment]
Yeah, I am pretty sure asyncStorage doesn't compress your data...
I don't think react-native-async-storage compresses data either
Well if you're looking for a way to compress your data asyncStorage is not really the option, I would suggest
